packages/jest-environment/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "jest-environment-hops",
+  "version": "14.0.0-nightly.4",
+  "description": "Opinionated Jest environment to run Puppeteer-powered tests against Hops apps",
+  "keywords": [
+    "hops",
+    "jest"
+  ],
+  "license": "MIT",
+  "engines": {
+    "node": "12 || 14 || 15"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/xing/hops.git",
+    "directory": "packages/jest-environment"
+  },
+  "main": "index.js",
+  "dependencies": {
+    "debug": "^4.1.1",
+    "eprom": "^1.0.0",
+    "fs-extra": "^9.0.1",
+    "mkdirp": "^1.0.4",
+    "mktemp": "^1.0.0",
+    "puppeteer": "^5.5.0",
+    "resolve-from": "^5.0.0",
+    "rimraf": "^3.0.2",
+    "url-join": "^4.0.1"
+  },
+  "devDependencies": {
+    "jest-environment-node": "^26.6.2"
+  },
+  "peerDependencies": {
+    "jest-environment-node": "^26.6.2"
```

Comment:
Why do we need the Jest environment to be a peer dependency?

Reply:
I thought I make consistent with all the other Jest-related packages being peer-dependencies.

Reply:
We only have those Jest dependencies as peer dependencies that the user actually needs in their own code.
For indirect dependencies that the user doesn't need to import, we have this in the Jest preset: https://github.com/xing/hops/blob/master/packages/jest-preset/jest-preset.js#L2-L18

Reply:
Yeah, I remember that one.